tests/setup.py

```diff
@@ -14,11 +14,8 @@
     include_package_data=True,
     install_requires=[
         'ducktape@git+https://github.com/vectorizedio/ducktape.git@master',
-        'prometheus-client==0.9.0',
-        'pyyaml==5.3.1',
-        'kafka-python==2.0.2',
-        'crc32c==2.2',
-        'confluent-kafka==1.6.1',
+        'prometheus-client==0.9.0', 'pyyaml==5.3.1', 'kafka-python==2.0.2',
```

Comment:
Was a formatter run on the setup.py file? If not lets just keep the original formatting, so the commit is an easy to understand one line change.

Reply:
Yep. Linter changes setup file(